app/components/finance/ecf/breakdown_summary.html.erb

```diff
@@ -74,15 +74,13 @@
 
 <table class="govuk-table payment-breakdown-table">
   <tbody>
-    <!--TODO: Add VAT-->
     <tr class="govuk-table__row">
       <th scope="row" class="govuk-table__cell govuk-!-font-weight-regular">VAT</th>
-      <td class="govuk-table__cell govuk-table__cell--numeric"><%= number_to_pounds 0 %></td>
+      <td class="govuk-table__cell govuk-table__cell--numeric"><%= number_to_pounds total_vat(@breakdown) %></td>
     </tr>
-    <!--TODO: Un-hardcode the values-->
     <tr class="govuk-table__row">
       <th scope="row" class="govuk-table__cell">Total payment</th>
-      <td class="govuk-table__cell govuk-table__cell--numeric govuk-!-font-weight-bold"><%= number_to_pounds 100_000 %></td>
+      <td class="govuk-table__cell govuk-table__cell--numeric govuk-!-font-weight-bold"><%= number_to_pounds total_payment(@breakdown) %></td>
```

Comment:
Should total payment include VAT?

Reply:
Good point, I wasn't sure it's already included or it's extra on top of total, I will check with Conor,